heat/core/tensor.py

```diff
@@ -876,45 +851,124 @@ def __ne__(self, other):
                 [1, 1]])
 
         """
-
         return relations.ne(self, other)
 
-
     def __pow__(self, other):
         """
-         Element-wise exponential function with values from second operand (scalar or tensor)
-         Takes the second operand (scalar or tensor) whose values are the exponent to be applied to the first
-         tensor as argument.
-
-         Parameters
-         ----------
-         other: tensor or scalar
-         The value(s) in the exponent (element-wise)
+        Element-wise exponential function with values from second operand (scalar or tensor)
+        Takes the second operand (scalar or tensor) whose values are the exponent to be applied to the first
+        tensor as argument.
 
-         Returns
-         -------
-         result: ht.tensor
-         A tensor containing the results of element-wise exponential operation.
+        Parameters
+        ----------
+        other: tensor or scalar
+            The value(s) in the exponent (element-wise)
 
-         Examples:
-         ---------
-         >>> import heat as ht
+        Returns
+        -------
+        result: ht.tensor
+            A tensor containing the results of element-wise exponential operation.
 
-         >>> T1 = ht.float32([[1, 2], [3, 4]])
-         >>> T1.__pow__(3.0)
-         tensor([[1., 8.],
-                 [27., 64.]])
+        Examples:
+        ---------
+        >>> import heat as ht
 
-         >>> T2 = ht.float32([[3, 3], [2, 2]])
-         >>> T1.__pow__(T2)
-         tensor([[1., 8.],
-                 [9., 16.]])
+        >>> T1 = ht.float32([[1, 2], [3, 4]])
+        >>> T1.__pow__(3.0)
+        tensor([[1., 8.],
+             [27., 64.]])
 
-         """
+        >>> T2 = ht.float32([[3, 3], [2, 2]])
+        >>> T1.__pow__(T2)
+        tensor([[1., 8.],
+             [9., 16.]])
 
+        """
         return arithmetics.pow(self, other)
 
+    def resplit(self, axis=None):
```

Comment:
This is not working for me locally.
I tried reproducing the example. The tensor.split does change, but lshape stays the same....